nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -54,15 +80,16 @@ def do_compression(
         ratio: float = None,
         group_size: int = None,
         all_layers: Optional[bool] = False,
+        activations: Optional[Dict[str, np.ndarray]] = None,
+        sensitivity_metric: Optional[SensitivityMetric] = SensitivityMetric.WEIGHT_QUANTIZATION_ERROR,
```

Comment:
`sensitivity_metric` and `all_layers` should be added to `dump_parameters`. What do you think about `dataset` (activations) parameter? Can we dump some information about it to IR?

Reply:
`ignored_scope` should be added as well

Reply:
Added `sensitivity_metric` and `all_layers`. `ignored_scope` is not available in this class. I'd not introduce additional parameter just for printing, rather than wait until refactoring https://github.com/openvinotoolkit/nncf/pull/2333/files#diff-0226e9a3e4f2dfdcc1950c3986f9c3c6c24ee5a996e95fda94bec70954637c2c
probably this parameter will be available